### DISCUSSION TITLE
gtk3 build with admin crashes with unprivileged user

### DISCUSSION THREAD

**Discussion by revelator** [msys2/MINGW-packages#12867] [Q&A]
I recently discovered that some tools built in an elevated bash prompt will only run if elevated afterwards, does bash inherit the windows permissions or what else could be going on ?

**Comment by Biswa96** [comment]
Please provide all the steps to reproduce the issue.

**Comment by revelator** [comment]
not much to it start mintty as admin build gtk3 result gtk3 can only be run as admin crashes if not.

**Reply by revelator** [reply]
guess im alone in having this behaviour ?

**Reply by sskras** [reply]
> start mintty as admin 
> build gtk3 
> result gtk3 can only be run as admin 
> crashes if not.

Now how do I build gtk3: using makepkg or just by downloading the official tarball ?

**Reply by revelator** [reply]
just use the PKGBUILD from here, not sure if it is still broken when built as admin though i dont remember noticing anything out of the ordinary with the recent package so should be fine. cd into mingw-w64-gtk3 and run makepkg-mingw -c -i --skippgp --nocheck and it should build.
